src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -2506,10 +2511,13 @@ def security_domain_upload(cmd, client, hsm_name, sd_file, sd_exchange_key, sd_w
         shared_keys=shared_keys,
         required=required
     )
+    return restore_blob_value
+
+def _security_domain_upload_blob(cmd, client, hsm_name, restore_blob_value, identifier=None, vault_base_url=None, no_wait=False):
     SecurityDomainObject = cmd.get_models('SecurityDomainObject', resource_type=ResourceType.DATA_PRIVATE_KEYVAULT)
     security_domain = SecurityDomainObject(value=restore_blob_value)
+    print(security_domain)
```

Comment:
Is this `print` for debug? Do you want to keep it in released CLI?

Reply:
Print is for debug, bugfix,  I removed it